lib/associations/mixin.js

```diff
@@ -68,6 +68,10 @@ const Mixin = {
     return this.getAssociations(target).find(association => this.verifyAssociationAlias(association, alias)) || null;
   },
 
+  getAssociationForForeignKey(foreignKey) {
+    return _.values(this.associations).find(association => association.foreignKey === foreignKey);
```

Comment:
No need to introduce this method to mixin, filter associations in `instance-validator`